sim/side.ts

```diff
@@ -918,6 +939,12 @@ export class Side {
 				if (data) return this.emitChoiceError(`Unrecognized data after "shift": ${data}`);
 				if (!this.chooseShift()) return false;
 				break;
+			case 'rotateright':
+			case 'rotateleft':
+				if (this.battle.gameType !== 'rotation') return this.emitChoiceError("Can't rotate: Not a rotation battle");
+				if (!data.startsWith('move ')) return this.emitChoiceError("Rotations must be followed by a move choice");
+				this.choice.willRotate = choiceType === 'rotateleft' ? 'left' : 'right';
+				return this.choose(data, true);
```

Comment:
This is one of those edge cases where my preference would be to fall through to the `move` choice (after adjusting the data appropriately) although my protocol option of having the rotation be a move flag bypasses this problem entirely.